### ISSUE TITLE
FlyBase Controlled Vocabulary BioPortal pull URL fails with 404 error

### ISSUE THREAD

**Issue by jvendetti** [FlyBase/flybase-controlled-vocabulary#98]
I'm one of the developers maintaining the [BioPortal application](http://bioportal.bioontology.org/). We have an entry for the [FlyBase Controlled Vocabulary](http://bioportal.bioontology.org/ontologies/FB-CV) that's configured to check the following URL nightly for new versions:

http://purl.obolibrary.org/obo/fbcv/dpo.owl

... which resolves to this URL:

https://raw.githubusercontent.com/FlyBase/flybase-controlled-vocabulary/master/dpo.owl

... which results in a 404 error.

I'm wondering if there's a new location for this particular ontology that we should be checking nightly? Thanks for any information you can provide.

**Comment by matentzn** [comment]
Yeah dpo moved to its own repo, check obofoundry registry. This purl should work:
http://purl.obolibrary.org/obo/dpo.owl :) let us if that works for you! Thanks!

**Comment by jvendetti** [comment]
Hi @matentzn. Thanks for that information. I noticed that we have a separate entry in BioPortal under the acronym DPO:

http://bioportal.bioontology.org/ontologies/DPO

... that pulls from the URL you listed above (http://purl.obolibrary.org/obo/dpo.owl). If I understand correctly then, our entry under the [acronym FB-CV](http://bioportal.bioontology.org/ontologies/FB-CV) is a duplicate with older metadata. Would it be acceptable to you if we removed it?
